adhocracy-plus/assets/js/app.js

```diff
@@ -31,11 +29,6 @@ function init () {
   ReactWidget.initialise('a4', 'ratings', ReactRatings.renderRatings)
   ReactWidget.initialise('a4', 'reports', ReactReports.renderReports)
 
```

Comment:
I think we should also remove Polls and possible reports from here too but as they come from a4 it's a little trickier, will add a pointless task